inference_perf/config.py

```diff
@@ -37,12 +37,17 @@ class LoadType(Enum):
     POISSON = "poisson"
 
 
-class LoadConfig(BaseModel):
-    type: LoadType = LoadType.CONSTANT
+class LoadStage(BaseModel):
```

Comment:
iiuc we lack the ability to send bursts of requests with this API, thoughts on making LoadStage abstract and creating stage subtypes for sustained load and bursts respectively?

Reply:
Yes that would be a different load type